tests/algorithms/test_centrality.py

```diff
@@ -156,3 +156,31 @@ def ratio(r, m, kind="CEC"):
         return 2 * r * (m - 1) / (np.sqrt(m**2 + 4 * (m - 1) * (r - 1)) + m - 2)
     elif kind == "HEC":
         return r ** (1.0 / m)
+
+
+def test_katz_centrality(edgelist1, edgelist3):
```

Comment:
As it currently stands, this function tests that all its contained code runs without errors, which is a good step in the right direction! We also need to include `assert` statements, as pytest specifically looks for those when running tests. It's always better to have more asserts than fewer. You can check out all the other test functions to see how asserts are handled elsewhere.

Reply:
Great!